doc/source/concept_and_workflow/systemdeps.rst

```diff
@@ -0,0 +1,96 @@
+.. _systemdeps:
+
+Host Requirements To Build Images
+---------------------------------
+
+When building OS images, several tools and sub-systems are used
+and required on the host {kiwi} is called at. For example to
+build a virtual disk image tools to create a partition table
+and also tools to create filesystems needs to be present on the
+host that builds the image.
```

Comment:
I think I know what you want to express, but the sentence looks a bit too long and complex to me. Could we simplify it?

One idea would be:

    ... to build a virtual disk images, several tools need to be available
    on the host that builds the image.
    This include tools for partition table or tools to create filesystems.